docs/docsite/rst/developers/contributing.rst

```diff
@@ -0,0 +1,467 @@
+        /var/lib/galaxy/venv/bin/python ./manage.py rebuild_galaxy_indexes
+        /var/lib/galaxy/venv/lib/python2.7/site-packages/maintenance/middleware.py:3: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
+        from django.utils.importlib import import_module
+
+        Rebuild Search Index
+        /var/lib/galaxy/venv/bin/python ./manage.py rebuild_index --noinput
+        /var/lib/galaxy/venv/lib/python2.7/site-packages/maintenance/middleware.py:3: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
+        from django.utils.importlib import import_module
+
+        Removing all documents from your index because you said so.
+        All documents removed.
+        Indexing 0 roles
+        + '[' 1 == 1 ']'
+        + scripts/docker/dev/sleep.sh
+
+   Once you see the very last line, ``scripts/docker/dev/sleep.sh``, 
+   you're ready for the next step.
+
+#. Now you'll start ``tmux`` and launch the processes inside the ``galaxy``
+   service container by running the following. If you're streaming the 
+   logging ouptut still, use ``<Ctrl-C>`` to stop the stream.
+
+   .. code-block:: shell-session
+
+        ## Set your working directory to the project root
+        $ cd galaxy
+
+        ## Start tmux
+        $ make dev/tmux
+
+Accessing the Galaxy web site
+-----------------------------
+
+After doing all this work, you're anxious to view your shiny new Galaxy site, 
+aren't you? Well, not so fast. As discussed above in [Running through tmux](#running-through-tmux), 
+you'll need to first check to make sure that all the database migrations 
+and setup work completed, and processes are running inside the ``galaxy`` 
+service container.
+
+Check the output stream from the ``galaxy`` service container, and look for 
+the completion of database migrations, and the start of the ``gulp`` web server.
+If you see output similar to the following, then you know that ``gulp`` is 
+running and accepting connections::
+
+    [03:10:00] Using gulpfile /galaxy/gulpfile.js
+    [03:10:01] Starting 'less'...
+    [03:10:01] Starting 'server'...
+    [HPM] Proxy created: /  ->  http://localhost:8888
+    [03:10:01] Finished 'server' after 130 ms
+    [03:10:01] Starting 'watch'...
+    [03:10:03] Finished 'watch' after 2.06 s
+    [Browsersync] Access URLs:
+     -----------------------------------
+           Local: http://localhost:8000
+        External: http://172.18.0.6:8000
+     -----------------------------------
+              UI: http://localhost:3001
+     UI External: http://172.18.0.6:3001
+     -----------------------------------
+    [Browsersync] Serving files from: /galaxy
+    [03:10:04] Finished 'less' after 3.23 s
+    [03:10:04] Starting 'default'...
+    [03:10:04] Finished 'default' after 108 μs
+
+OK, go for it! Your Galaxy web site is available at: http://localhost:8000.
+
+.. note:: You won't be able to authenticate until you perform the post build 
```

Comment:
Might be wrong, but think there should be a line break after `.. note::`

Reply:
Nope, there is no break required.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions